go.mod

```diff
@@ -304,5 +304,6 @@ replace (
 	github.com/dgrijalva/jwt-go => github.com/form3tech-oss/jwt-go v3.2.6-0.20210809144907-32ab6a8243d7+incompatible
 	github.com/go-ldap/ldap/v3 => github.com/YangKeao/ldap/v3 v3.4.5-0.20230421065457-369a3bab1117
 	github.com/pingcap/tidb/parser => ./parser
+	github.com/tikv/client-go/v2 => github.com/SpadeA-Tang/client-go/v2 v2.0.1-0.20230703094423-9f0b330b9fad
```

Comment:
Please provide the url of the corresponding PR of client-go, and ask for reviews for that PR first.

Reply:
https://github.com/tikv/client-go/pull/865